plugins/techdocs/src/reader/transformers/sanitizeDOM.ts

```diff
@@ -14,11 +14,32 @@
  * limitations under the License.
  */
 
+const TECHDOCS_CSS = /main\.[A-Fa-f0-9]{8}\.min\.css/;
+const GOOGLE_FONTS = /fonts\.googleapis\.com/;
+const GSTATIC_FONTS = /fonts\.gstatic\.com/;
+
+export const safeLinksHook = (node: Element) => {
+  if (node.nodeName && node.nodeName === 'LINK') {
+    const href = node.getAttribute('href') || '';
+    if (href.match(TECHDOCS_CSS)) {
```

Comment:
I'm wondering here if the regex is too forgiving? Do we maybe want to add anchors so we can check that the href starts with `https:///fonts.googleapis.com` instead? Thinking that an attacker could just have `http://badwebsite.com/fonts.googleapis.com.css` or whatever that could be bad? not sure what this means and if its even possible, it's just I can see that it could be matched in many different ways not just the happy path?

Reply:
Good catch, thank you, suggestion applied!